CRITICAL - 2024-05-20 11:24:05 --> ErrorException: Attempt to read property "id" on array
[Method: GET, Route: liste]
in APPPATH\Views\liste.php on line 25.
 1 APPPATH\Views\liste.php(25): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "id" on array', 'C:\\xampp\\htdocs\\project-root\\-connectify\\app\\Views\\liste.php', 25)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\project-root\\-connectify\\app\\Views\\liste.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('liste', [], true)
 5 APPPATH\Controllers\Personnes.php(15): view('liste', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Personnes->liste()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personnes))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 11:24:09 --> ErrorException: Attempt to read property "id" on array
[Method: GET, Route: liste]
in APPPATH\Views\liste.php on line 25.
 1 APPPATH\Views\liste.php(25): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "id" on array', 'C:\\xampp\\htdocs\\project-root\\-connectify\\app\\Views\\liste.php', 25)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\project-root\\-connectify\\app\\Views\\liste.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('liste', [], true)
 5 APPPATH\Controllers\Personnes.php(15): view('liste', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Personnes->liste()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personnes))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 11:24:38 --> ErrorException: Attempt to read property "id" on array
[Method: GET, Route: liste]
in APPPATH\Views\liste.php on line 25.
 1 APPPATH\Views\liste.php(25): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "id" on array', 'C:\\xampp\\htdocs\\project-root\\-connectify\\app\\Views\\liste.php', 25)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\project-root\\-connectify\\app\\Views\\liste.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('liste', [], true)
 5 APPPATH\Controllers\Personnes.php(15): view('liste', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Personnes->liste()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Personnes))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
ERROR - 2024-05-20 21:35:23 --> mysqli_sql_exception: Unknown column 'username' in 'where clause' in C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\project-root\-connectify\app\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Login->authenticate()
#8 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\project-root\-connectify\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-20 21:35:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'username' in 'where clause'
[Method: POST, Route: login/authenticate]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->authenticate()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:35:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'username' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->authenticate()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:35:23 --> [Caused by] mysqli_sql_exception: Unknown column 'username' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->authenticate()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
ERROR - 2024-05-20 21:37:39 --> mysqli_sql_exception: Unknown column 'username' in 'where clause' in C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\project-root\-connectify\app\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Login->authenticate()
#8 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\project-root\-connectify\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\project-root\-connectify\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-05-20 21:37:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'username' in 'where clause'
[Method: POST, Route: login/authenticate]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->authenticate()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:37:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'username' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->authenticate()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:37:39 --> [Caused by] mysqli_sql_exception: Unknown column 'username' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `username` = \'test@test.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\Login.php(33): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->authenticate()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:38:36 --> Error: Class "App\Models\LoginModel" not found
[Method: POST, Route: login/authenticate]
in APPPATH\Controllers\Login.php on line 32.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->authenticate()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:01 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:03 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:07 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:36 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:37 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:37 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:38 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:49:53 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:51:21 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
CRITICAL - 2024-05-20 21:51:22 --> Error: Call to undefined function set_value()
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 26.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 4 APPPATH\Controllers\Login.php(12): view('login')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Login->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project-root\\-connectify\\public\\index.php')
